source/whats-new.txt

```diff
@@ -58,6 +58,24 @@ What's New in 6.0
      import these methods from the driver instead of from
      ``mongodb-client-encryption``.
 
+   - Removed support for strings with a length of 12 from the ``ObjectId``
+     constructor.
+
+   - The ``abortTransaction`` and ``commitTransaction`` methods return
+     ``null`` instead of the raw command results.
+
+   - Removed connection option helpers that recognized values other than ``true``
+     or  ``false`` as booleans. You must provide either ``true`` or ``false`` values for
+     options that expect boolean values in the connection string or MongoClient
+     constructor.
+
+   - Removed the Binary BSON type constructor that accepted a string.
+
+   - The ``Binary.write`` method no longer accepts a string to write
+     to the binary BSON object.
```

Comment:
I'm not sure if the suggested change is correct since they didn't remove the method that accepted the string. They just removed string as one of the accepted parameters to that method